libs/npm/core/src/hooks/api-editor/interfaces/IUserInfoModel.ts

```diff
@@ -11,6 +11,7 @@ export interface IUserInfoModel {
   lastLoginOn?: Date;
   isEnabled: boolean;
   status: UserStatusName;
+  preferences?: any;
```

Comment:
This is currently incorrect.  The `IUserInfoModel` does not contain this property.